clojurebridge_attendee_care.md

```diff
@@ -0,0 +1,39 @@
+# Attendee care recipe
+
+`This is our recepe to attendee care. The tasks can be split. Different people can take over parts.`
+
+**Selection:**
+
+1. In our GoogleDrive:
```

Comment:
maybe 0: We have a Google account you can receive access to from previous organizers. There you will find attendee documents and email correspondence.